nherriot/noisy-atom-portal
# 45 remove all models that are not used on the system

To test this:

1. Go to branch: 45-remove-all-models-that-are-not-used-on-the-system
2. Git pull.
3. Run server in **dev** mode.
  ```
     /> cd app/
     /> export ENVIRONMENT=dev
     /> python manage.py runserver 8005
4. Make sure the directories xxx are missing.
5. 

- Delete all models of accounts, catalog and qrecode all those codes are not Django vanilla.
- Removed all migrations and run migrate to make sure all is working and one single migration.
- Removed admin of those app models are delete.
-  Django admin working OK.
- Blog Add, list and detail are working on admin and when the user is logged in if use the buttons. 
- Update blogs has bug and fixed as well.

I am just wondering if the same models of above are required as Django vanilla (Django rest framework)?
